price/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "price"
+version = "0.1.0"
+description = "Price Oracle for the Helium Network"
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+
+[dependencies]
+anyhow = {workspace = true}
+config = {workspace = true}
+clap = {workspace = true}
+thiserror = {workspace = true}
+serde =  {workspace = true}
+serde_json = {workspace = true}
+tonic = {workspace = true}
+futures = {workspace = true}
+futures-util = {workspace = true}
+prost = {workspace = true}
+tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
+metrics = {workspace = true }
+metrics-exporter-prometheus = { workspace = true }
+tokio = { workspace = true }
+chrono = { workspace = true }
+helium-proto = { workspace = true }
+file-store = { path = "../file_store" }
+poc-metrics = { path = "../metrics" }
+triggered = {workspace = true}
+pyth-sdk-solana = "0.7.1"
+solana-client = ">= 1.9, < 1.15"
+solana-program = ">= 1.9, < 1.15"
```

Comment:
Do you need all of these?

Reply:
Yeah, client for the RpcClient and program for the pubkey. pyth-sdk doesn't seem to automatically fetch these.

Reply:
Removing just the solana-program doesn't seem to work either.